cypress.config.js

```diff
@@ -15,7 +15,7 @@ module.exports = defineConfig({
           const randomNumber = Math.ceil(Math.random(1000) * 1000);
           return {
             username: faker.name.firstName() + `${randomNumber}`,
-            email: 'test' + `${randomNumber}` + '@mail.com',
```

Comment:
Your email addresses should also be unique for each user. Consider using faker to generate unique email addresses.